test_common/gles/gl_headers.h

```diff
@@ -40,21 +39,33 @@
 #define glDeleteFramebuffersEXT      glDeleteFramebuffers
 #define glBindFramebufferEXT         glBindFramebuffer
 #define glFramebufferRenderbufferEXT glFramebufferRenderbuffer
+
+#ifndef GL_ES_VERSION_3_0
 #define glTexImage3D                 glTexImage3DOES
+#define GL_RGBA32F GL_RGBA32F_EXT
+//#define GL_TEXTURE_3D                GL_TEXTURE_3D_OES
+#define GL_READ_ONLY GL_BUFFER_ACCESS_OES
+#define GL_HALF_FLOAT_ARB GL_HALF_FLOAT_OES
+#define GL_BGRA GL_BGRA_EXT
+#define glMapBufferRange glMapBufferRangeEXT
+#define glUnmapBuffer glUnmapBufferOES
+#else
+//#define GL_HALF_FLOAT_OES            GL_HALF_FLOAT
+//#define GL_RGBA8_OES                 GL_RGBA8
```

Comment:
Why are there commented-out definitions?

Reply:
have defined in GLES2/gl2ext.h

Reply:
@ijkmn  I have build errors on ubuntu 20.04. Could you check on your side ?
test_fence_sync.cpp:300:24: error: cast to pointer from integer of different size [-Werror=int-to-pointer-cast]
  300 |         return (void *)error;

`
/home/gta/gwa/OpenCL-CTS/test_conformance/gles/test_fence_sync.cpp: In member function ‘virtual void* RunThread::IRun()’:
/home/gta/gwa/OpenCL-CTS/test_conformance/gles/test_fence_sync.cpp:300:24: error: cast to pointer from integer of different size [-Werror=int-to-pointer-cast]
  300 |         return (void *)error;
      |                        ^~~~~
In file included from /usr/include/string.h:495,
                 from /home/gta/gwa/OpenCL-CTS/test_conformance/gles/testBase.h:22,
                 from /home/gta/gwa/OpenCL-CTS/test_conformance/gles/test_fence_sync.cpp:18:
In function ‘char* strcpy(char*, const char*)’,
    inlined from ‘GLuint createShaderProgram(GLint*, GLint*)’ at /home/gta/gwa/OpenCL-CTS/test_conformance/gles/test_fence_sync.cpp:164:11:
/usr/include/x86_64-linux-gnu/bits/string_fortified.h:90:33: error: ‘void* __builtin___memcpy_chk(void*, const void*, long unsigned int, long unsigned int)’ writing 76 bytes into a region of size 75 overflows the destination [-Werror=stringop-overflow=]
   90 |   return __builtin___strcpy_chk (__dest, __src, __bos (__dest));
      |          ~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1plus: all warnings being treated as errors
`

Reply:
have fixed